packages/sdk/src/Actions/EventAction.ts

```diff
@@ -322,6 +323,19 @@ export interface EventActionPayloadRaw {
   actionStepFour: ActionStep;
 }
 
+type ReadEventActionParams<
+  fnName extends ContractFunctionName<typeof eventActionAbi, 'pure' | 'view'>,
+> = ReadParams<typeof eventActionAbi, fnName>;
+
+type TxParams = ValidateEventStepParams | ValidateFunctionStepParams;
+
+type ValidateInputParams<
+  fnName extends ContractFunctionName<typeof eventActionAbi, 'pure' | 'view'>,
+> = {
+  getterParams?: ReadEventActionParams<fnName>;
```

Comment:
nit: think this name can be more specific to the operation it's being applied to, maybe `getActionStepsParams`?